app/Http/Controllers/ProfileController.php

```diff
@@ -84,4 +85,33 @@ public function disable2fa(Request $request)
 
         return redirect()->route('profile.index')->with('success', trans('messages.profile.2fa.disabled'));
     }
+
+    public function searchUsers(Request $request)
+    {
+        return User::select('id', 'name')->where('name', 'LIKE', '%'.$request->q.'%')->get();
+    }
+
+    public function transferMoney(Request $request)
+    {
+        $receiver = User::where('name', $request->input('name'))->first();
+        if (! $receiver) {
+            return redirect()->route('profile.index')->with('error', trans('messages.not-authorized'));
+        }
```

Comment:
We can just use the validator below for this, and the error message would be clearer